AdyenComponents/PayTo/PayToPayIdentifier.swift

```diff
@@ -4,24 +4,26 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
 /// Identifier options for PayTo, known as PayID.
-internal enum PayToPayIdentifier: String, CustomStringConvertible, CaseIterable {
```

Comment:
Is the compiler not complaining at the removal of `CustomStringConvertible` or is it added somewhere else?

Reply:
No complains.